server/src/routes/github.js

```diff
@@ -0,0 +1,26 @@
+import express from 'express';
+import { githubService } from '../service';
+
+const router = express.Router();
+
+const parseGithubData = (payload) => {
+  const data = payload.data;
+  return ({
+    status: 'success',
+    data: {
+      previous: 5,
+      current: data.length,
+      updated: new Date(),
+    },
+  });
+};
+
+router.get('/pr', (req, res, next) => {
```

Comment:
Please utilize our caching system with createCachedRouterCallback. See e.g. the jira routes.